config/contents/style/block_delimiters.md

```diff
@@ -100,3 +100,29 @@ multi-line blocks.
     words.each { |word|
       word.flip.flop
     }
+
+### Example: BracesRequiredMethods: ['sig']
+
+    # Methods listed in the BracesRequiredMethods list, such as 'sig'
+    # in this example, will require `{...}` braces. This option takes
+    # precedence over all other configurations except IgnoredMethods.
+
+    # bad
+    sig do
+      params(
+        foo: string,
```

Comment:
Trailing comma, is this a deliberate choice?

Reply:
This appears to be [a choice made upstream](https://github.com/rubocop-hq/rubocop/blob/v0.80.0/lib/rubocop/cop/style/block_delimiters.rb#L119), whether a typo or deliberate I cannot say.